src/HotChocolate/Core/src/Types.Scalars/WellKnownScalarTypes.cs

```diff
@@ -5,5 +5,7 @@ internal static class WellKnownScalarTypes
         public const string NonEmptyString = nameof(NonEmptyString);
         public const string PositiveIntType = nameof(PositiveIntType);
```

Comment:
This is something i messed up in another PR. 
Do you mind cleaning this up? 
```
        public const string PositiveInt = nameof(PositiveInt);
```

This will bring some test to fail :) you will see